.github/workflows/release-integration.yml

```diff
@@ -39,46 +39,32 @@ jobs:
           name: nri-kubernetes-${{ matrix.goos }}-${{ matrix.goarch }}
           path: bin/nri-kubernetes-${{ matrix.goos }}-${{ matrix.goarch }}
 
-  release:
-    name: Release docker images and config
+  docker-release:
+    name: Release docker images
     needs: [ build ]
     runs-on: ubuntu-latest
     env:
       DOCKER_IMAGE_NAME: newrelic/nri-kubernetes
       DOCKER_PLATFORMS: "linux/amd64,linux/arm64,linux/arm" # Must be consistent with the matrix from the job above
     steps:
-      - name: Generate docker image version from git tag
-        run: |
-          echo "${{ github.event.release.tag_name }}" | grep -E '^v[0-9.]*[0-9]$'
-          DOCKER_IMAGE_TAG=$(echo "${{ github.event.release.tag_name }}" | sed 's/^v//')
-          echo "DOCKER_IMAGE_TAG=$DOCKER_IMAGE_TAG" >> $GITHUB_ENV
-
-      - if: ${{ github.event.release.prerelease }}
-        run: |
-          echo "DOCKER_IMAGE_TAG=${DOCKER_IMAGE_TAG}-pre" >> $GITHUB_ENV
-          echo "S3_PATH=$S3_PATH/test" >> $GITHUB_ENV
```

Comment:
Docker `-pre` suffix has been moved to a step below. What about this `S3_PATH`?

Reply:
S3_PATH is not useful anymore, since we are not publishing things in S3